javascript/javascript1/week3/showStatus.js

```diff
@@ -0,0 +1,36 @@
+const activities = [];
+let usageLimit = 120; // in minutes
+
+function addActivity(date, activity, duration) {
+  const newActivity = {
+    date: date || new Date().toLocaleDateString(),
+    activity: activity,
+    duration: duration,
+  };
+  activities.push(newActivity);
+}
+
+function showStatus() {
+  if (activities.length === 0) {
+    console.log("Add some activities before calling showStatus");
+    return;
+  }
+
+  let totalDuration = 0;
+  activities.forEach(activity => {

```

Comment:
This hand-in has a mix of for-loops and more advanced features like `forEach`. Are there any particular reason for the switch back and forth? 😄

Also can you briefly explain what `forEach`and `=>` does? 🙂

Remember to only move onto functions like `forEach` once you are comfortable with for loops and higher-order functions! If you already are, then I'd recommend looking into reduce for this case as it is more fitting. 🧠